ui/src/components/Read/index.vue

```diff
@@ -197,11 +195,10 @@ function initCode() {
     } else {
         let timeRange = {
             timeRange: {
-                begin: new Date(new Date() - 15),
+                begin: new Date(new Date() - 15 * 60 * 1000),
```

Comment:
This is a good find.

Since we have a constant variable named 'last15Minutes' defined at line 35, would it be possible to reuse it instead of generating it repeatedly? I understand that this constant is a "ref", but it would be better to use a numeric constant instead and refer to it in the two places it is used.

Reply:
The "ref" here means reactive reference. When the value of the ref changes, any Vue components that depend on this value will automatically re-render to reflect the new value. 

I think we should not use "ref" here for these time variables (last15Minutes, lastMonth, ...).  
We should just change to `const last15Minutes = 900 * 1000`.

Reply:
Changed accordingly.